resources/rulefiles/README.md

```diff
@@ -0,0 +1,3 @@
+# Rulefile Samples
+
+Collection of rulefiles for ease of access. These files are not guaranteed to be valid
```

Comment:
Avoid using the word collection.
"Some samples of rulefiles for ease of access. (...)"

Reply:
Replaced